DelfinPerformance/README.md

```diff
@@ -0,0 +1,67 @@
+## Delfin Performance Monitoring with Preometheus
+
+This is demo of SODA Foundations Delfin performance monitoring feature using default preometheus exporter.
+
+
+Delfin is a heterogeneous infrastructure management platform. It manages and monitors multiple storage backends from a single place. Prometheus is default integrated with delfin for monitoring performances of the storages.
+
+The detailed analysis and architecture of prometheus integration with delfin can be found [here](https://github.com/sodafoundation/design-specs/blob/dcdee7b67d4a4ee74f065f00b2e93efb22f2493a/specs/SIM/PerfomanceMontoringDesign.md)
+
+Follow the below steps to setup delfin and prometheus or go through the demo [video]() for the reference.
+
+step1: Install and start prometheus server
+
+1. [Download the latest binaries from here](https://prometheus.io/download/) and run the below steps.
+
+  ```sh
+  1. tar xvfz prometheus-*.tar.gz
+
+  2. cd prometheus-*
+
+  3. ./prometheus
+  ```
+
+2. Example:
+```sh
+root@root:/prometheus/prometheus-2.20.0.linux-amd64# ./prometheus
+```
+
+3. Install and run the Delfin from [here](https://docs.sodafoundation.io/soda-gettingstarted/installation-using-ansible/)
```

Comment:
Why dont we give specific link only to delfin? Those steps will take the user to soda right?
Needed?

Reply:
we can give link to stand alone install doc, 
https://github.com/sodafoundation/delfin/blob/master/installer/README.md

Reply:
done